### DISCUSSION TITLE
User Management-API: same code as database-API but second instance on vercel.com?

### DISCUSSION THREAD

**Discussion by ghost** [technologiestiftung/giessdenkiez-de#396] [Q&A] [answered]
Hi folks,

I'm trying to set up gieß-den-kiez for my city but do not quite understand the[ "User Management-API"-Setup](https://github.com/technologiestiftung/giessdenkiez-de/wiki/Backend-%E2%80%94-Setup-Your-User-Management-API)

As far as I understand it, I create a second vercel project with the gdk-api code but with more Auth0-variables.

Right?

Thank you very much in advance!

PS: I was hoping to find a tech-channel on the giess-den-kiez-slack but there does not seem to be one. Is there a more elegant way to get in contact with you?

**Comment by ff6347** [comment, marked as answer]
Hi @TomThats 

The user management is indeed a second deployment.

> As far as I understand it, I create a second vercel project with the gdk-api code but with more Auth0-variables.

The source is located in the repo https://github.com/technologiestiftung/tsb-trees-api-user-management/ (sorry for the bad naming, we did not find the time for a rename+redeploy). It is **not** the main API that connects to the database.

This API is only for communicating with the Auth0 management API. It is used to allow users to remove their accounts.

**Reply by ff6347** [reply]
Here are the Auth0 API docs https://auth0.com/docs/api/management/v2
